### PR TITLE
Add spacing between inputs in Investor SAFT setup Step 4

### LINKED ISSUE #574: Fix spacing in Investor ID card (Step 4)

### PR DESCRIPTION
**Pull request by as-dr**
Added `16px` margin-top. 

Need to wait until the Ledger issues are fixed for me to eyeball it & confirm it's good.

### CODE REVIEW

File: components/Investor/Home.jsx [hunk 1]
```diff
@@ -91,25 +91,19 @@ const PreConfirm = ({ address, investor, path }) => {
...
+          justifyContent='flex-end'
+          height={4}
           width='100%'
+          mb={2}
         >
           <BaseButton
-            mt={3}
-            m='0'
-            p='0'
+            // m='0'
```
Comment (Schwartz10) [inline components/Investor/Home.jsx]: @as-dr just delete things, no comments

**Review by Schwartz10 — CHANGES_REQUESTED** [review]
Just delete those comments and ping me when you make that commit

File: components/Investor/Home.jsx (hunk 1 above)
Comment (as-dr) [reply components/Investor/Home.jsx]: @Schwartz10 Brain fart. Done.

**Comment by as-dr** [comment]
@Schwartz10 Good to go

**Review by Schwartz10 — APPROVED** [review]

**Pull request merged**